src/ViewTrait.php

```diff
@@ -470,7 +501,7 @@ public function renderFile(string $viewFile, array $parameters = []): string
      */
     public function localize(string $file, ?string $language = null, ?string $sourceLanguage = null): string
     {
-        $language = $language ?? $this->state->getLanguage();
+        $language = $language ?? $this->language ?? $this->state->getLanguage();
```

Comment:
Too implicit behavior. Suggest remove `withLanguage()` and leave only new parameter "language" in methods.

Reply:
I agree with @vjik as this is only used for rendering which already has such a parameter.